rbac/init.go

```diff
@@ -124,6 +127,8 @@ func Init(adminUserID string) error {
 		{RoleResponder, ObjectDatabaseIncident, ActionUpdate},
 
 		{RoleViewer, ObjectDatabase, ActionRead},
+
+		{RoleAgent, ObjectAgentPush, ActionWrite},
```

Comment:
The agent should only have access to call the API that would be used when pushing upstream

Reply:
Isn't it already that way?